bodhi/server/models.py

```diff
@@ -738,6 +748,8 @@ class Build(Base):
             of.
         type (int): The polymorphic identify of the row. This is used by sqlalchemy to identify
             which subclass of Build to use.
+        ci_status (EnumSymbol): The CI status of the update. This must be one of the values
+            defined in :class:`CiStatus` or ``None``.
```

Comment:
It might be good to document that ```None``` is used for Updates that were created when the CI integration was not enabled.